lib/exec.js

```diff
@@ -139,6 +139,41 @@ function executeDirect(command, args, options, filenameTransform) {
     });
 }
 
+function withNsjailTimeout(args, options) {
+    if (options && options.timeoutMs) {
+        const ExtraWallClockLeewayMs = 1000;
+        return args.concat([
+            '--time_limit',
+            `${Math.round((options.timeoutMs + ExtraWallClockLeewayMs) / 1000)}`
+        ]);
+    }
+    return args;
+}
+
+function sandboxNsjail(command, args, options) {
+    logger.info("Sandbox execution via nsjail", {command, args});
+    const execPath = path.dirname(command);
+    const execName = path.basename(command);
+
+    const jailingOptions = withNsjailTimeout([
+        '--config', 'etc/nsjail/sandbox.cfg',
+        '--cwd', '/app',
+        '--bindmount', `${execPath}:/app`,
```

Comment:
Cunning; this is how we map in the sandbox. Nice that there's already a `--cwd` :)